syntax = "proto3";

message TimeRequest {
  int64 client_time = 1;
}

message TimeResponse {
  int64 server_time = 1;
}

service ClockSync {
  rpc GetTime(TimeRequest) returns (TimeResponse);
  rpc UpdateTime(TimeRequest) returns (TimeResponse);
}
